rate.go

```diff
@@ -158,6 +158,15 @@ func (l Limiter) AllowAtMost(
 	return res, nil
 }
 
+// Reset gets a key and reset all limitations and previous usages
+func (l *Limiter) Reset(ctx context.Context, key string) error {
+	_, err := reset.Run(ctx, l.rdb, []string{redisPrefix + key}, nil).Result()
```

Comment:
Please remove Lua script and just call `l.rdb.Del(ctx, redisPrefix + key)`

Reply:
Thanks for the comment. I just removed the script and used the Del function instead.